ostree-ng.sh

```diff
@@ -334,11 +334,9 @@ description = "A base rhel-edge container image"
 version = "0.0.1"
 modules = []
 groups = []
-
```

Comment:
You did not define `EMBEDDED_CONTAINER` in this shell script.

Reply:
I did not define it because I think we can reuse the behavior of the variable CONTAINER_PUSHING_FEAT.
Nevertheless I will apply the rest of suggested corrections first.

Reply:
Yeah, but CONTAINER_PUSHING_FEAT does not make sense for this feature. It'll be great to define a new variable for this feature only. Thanks.

Reply:
Ok! No problem, I will work on that.